contracts/domain/BosonTypes.sol

```diff
@@ -65,6 +65,8 @@ contract BosonTypes {
 
     enum AuthTokenType {
         None,
+        // For future use
+        Custom,
```

Comment:
Maybe put this on the same line like

`Custom, // For future use`

so it doesn't look like the whole list from None down is for future use